lib/active_yaml/base.rb

```diff
@@ -24,7 +28,9 @@ def load_path(path)
       end
```

Comment:
Sigh. and then there was one...

looks like the last failure is related to Psych >= 4.0

```ruby
        result = File.read(path)
        result = ERB.new(result).result if process_erb
        YAML.unsafe_load(result)
```

Appreciate your patience especially since you are no longer using this library.